plugin.video.retrospect/resources/libs/helpers/logsender.py

```diff
@@ -0,0 +1,319 @@
+# ===============================================================================
```

Comment:
not allowed, please call xbmc log uploader and the ultimate decision has to come from the user

Reply:
This is the retrospect.log, not kodi.log and this is a method that is not use automatically. The user has to select the "Upload log" option from the add-on settings. It does upload it to paste.kodi.tv.